DavaiPosmotrim/WebSockets/WebSocketsAPI.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  WebSocketsAPI.swift
+//  DavaiPosmotrim
+//
+//  Created by Эльдар Айдумов on 15.09.2024.
+//
+
+import Foundation
+
+enum Endpoint {
+    case usersUpdate
+    case sessionStatusUpdate
+    case moviesMatchesUpdate
+    case rouletteUpdate
+    case sessionResultUpdate
+}
```

Comment:
Предлагаю путь эндпонинта добавить как переменную в само перечисление, например вот так:
```suggestion

    var urlPath: String {
        switch self {
        case .usersUpdate: "/users/"
        case .sessionStatusUpdate: "/session_status/"
        case .moviesMatchesUpdate: "/matches/"
        case .rouletteUpdate: "/roulette/"
        case .sessionResultUpdate: "/session_result/"
        }
    }
 }
```

Reply:
Done